webpack/assets/javascripts/compute_resource/libvirt.js

```diff
@@ -54,26 +56,38 @@ export function imageSelected(item) {
 
   if (template) {
     const url = $(item).attr('data-url');
+    // For some reason there are two help blocks
+    // so we need to select the correct one
+    const help = $('#image_selection .form-group > div > .help-block');
```

Comment:
In which situation are there two?

Reply:
AFAIK there are always two

Reply:
I don't know why there are two of them. Somewhere, deep in the helper's helper methods, the help text is generated.

Reply:
Can you inspect both and screen shot them? I only see one without this fix

Reply:
![help-block](https://github.com/theforeman/foreman/assets/59385976/fbd22f99-d8aa-4041-90d0-60961eea32a5)

For `#image_selection .help-block` selector there are two blocks.